runtime/src/lib.rs

```diff
@@ -1041,7 +1041,10 @@ impl_runtime_apis! {
 
     impl sp_block_builder::BlockBuilder<Block> for Runtime {
         fn apply_extrinsic(extrinsic: <Block as BlockT>::Extrinsic) -> ApplyExtrinsicResult {
-            Executive::apply_extrinsic(extrinsic)
+            Executive::apply_extrinsic(extrinsic).map_err(|err| {
+                frame_support::debug::error!(target: xpallet_support::RUNTIME_TARGET, "[apply_extrinsic]|{:?}", err);
```

Comment:
Very happy to see we could remove this patch. I actually would suggest:
```suggestion
                frame_support::debug::error!(target: xpallet_support::RUNTIME_TARGET, "Extrinsic failed: {:?}", err);
```

but it's up to you, I'm ok with both.

Reply:
To be honest, I don’t know why we need a patch version. Compared with the upstream version, it only adds a line of log in the `apply_extrinsic` function.